lib/reporters/bower/index.js

```diff
@@ -0,0 +1,53 @@
+'use strict';
+
+var fs          = require( 'fs' );
+var path        = require( 'path' );
+var creditUtil  = require( '../../../util/credit' );
+var packageUtil = require( '../../../util/package' );
+
+
+/**
+* Read project root and evaluate dependency credits for bower modules
+*
+* @param  {String} projectPath      absolute path to project root
+*
+* @return {Array}                   list of credits
+*/
+function getBowerCredits( projectPath ) {
```

Comment:
I think `getCredits` is enough for the analyzers. No need to repeat the specific package manager in function names, or what do you think?